### PR TITLE
Export map

### PR DESCRIPTION
**Pull request by gfiorav**
@csobier please review the texts (all but the error text are the same as last time).

@urbanphes please review the design

Confirm:
screen shot 2016-07-04 at 13 49 24

Click to download (if popups blocked):
screen shot 2016-07-04 at 13 56 57

Error:
screen shot 2016-07-04 at 13 49 01

Flow (first Ok, next with error):

new_export

### CODE REVIEW

**Comment by gfiorav** [comment]
I've added some tests, please CR @xavijam

**Comment by saleiva** [comment]
VAMOS NENO

File: lib/assets/javascripts/cartodb3/editor/components/modals/export-map/modal-export-map-view.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    }, this);
+  },
+
+  _disableBinds: function () {
+    $(document).unbind('keydown', this._onKeyDown);
+  },
+
+  _onKeyDown: function (ev) {
+    var keyCode = ev.which;
+    if (keyCode === $.ui.keyCode.ENTER) {
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/components/modals/export-map/modal-export-map-view.js]:
```js
var ENTER_KEY_CODE =  13;
if (keyCode === ENTER_KEY_CODE) {
…
}
```

**Comment by xavijam** [comment]
🇪🇸

**Comment by gfiorav** [comment]
changed that @xavijam ! tomorrow I'll deploy if no other comments popup

**Comment by xavijam** [comment]
Check travis errors :)

**Pull request merged**